tests/test_modeling_speech_encoder_decoder.py

```diff
@@ -382,10 +372,23 @@ def test_real_model_save_load_from_pretrained(self):
 
 @require_torch
 class Wav2Vec2BertModelTest(EncoderDecoderMixin, unittest.TestCase):
-    def get_pretrained_model(self):
-        return SpeechEncoderDecoderModel.from_encoder_decoder_pretrained(
+    def get_pretrained_model_and_inputs(self):
```

Comment:
Only `Wav2Vec2BertModelTest.test_real_model_save_load_from_pretrained` actually gets executed. Other models have no suitable pretrained encoders or decoders for now.